node/task.ts

```diff
@@ -1797,11 +1798,18 @@ export function getHttpProxyConfiguration(requestUrl?: string): ProxyConfigurati
             return null;
         }
         else {
+            const parsedUrl: URL = new URL(proxyUrl);
+            let proxyAddress: string = `${parsedUrl.protocol}//${parsedUrl.host}`;
+            if (proxyUsername) {
+                proxyAddress = `${parsedUrl.protocol}//${proxyUsername}:${proxyPassword}@${parsedUrl.host}`;
+            }
+            console.log('returning proxy config');
```

Comment:
Do we really need to signalize about it in stdout (not even on debug level)?

Reply:
removed